portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,53 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
   @Override
+  public void init() {
+    messages = new ArrayList<>();
+    messages.add("lol");
+    messages.add("lmao");
+    messages.add("haha");
+    /**String json = convertToJson(messages);
+    response.getWriter().println(json);*/
+  }
+
+  private String convertToJson(List<String> messages) {
+    String json = "{";
+    for (int i = 0; i < messages.size()-1; i++) {
+        json += messages.get(i) + ", ";
+    }
+    json += messages.get(messages.size()-1);
```

Comment:
You can use `String.join` here instead of adding the messages one-by-one.

https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-